sdcm/sct_events/base.py

```diff
@@ -190,6 +205,36 @@ def timestamp(self) -> float:
         """
         return self.source_timestamp or self.event_timestamp
 
+    @property
+    def msgfmt(self):
+        fmt = "({0.base} {0.severity}) period_type={0.period_type} event_id={0.event_id}"
+
+        during_nemesis = []
+        for event_context in self.subcontext:
+            if event_context["base"] != "DisruptionEvent":
+                continue
+
+            during_nemesis.append(event_context['nemesis_name'])
+
+        if during_nemesis:
+            fmt += f" during_nemesis={','.join(during_nemesis)}"
```

Comment:
why not call it `context=` ? 

and put the creating of the context in it's own function  `contextfmt` ? where you can extend it and make it handle more types of nemesis ?